### PR TITLE
Hardware track

### PR DESCRIPTION
**Pull request by smohamu3**
I forgot to mention that I added the links to the registration site hehe

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -197,6 +197,13 @@ <h3>Beginner</h3>
                 <p>New to hackathons? No worries! You'll be provided with dedicated resources to help you be successful at Technica and beyond! This includes everything from an introduction to hackathons and Technica to universal coding and logic concepts!</p>
               </div>
             </div>
+            <div class="col-lg">
+              <div class="hardware" style="height: 100%">
+                <div></div>
+                <h3>Hardware</h3>
+                <p>Want to spark up your project with some hardware? Great news! At Technica, we’ll provide you with the resources for you to kickstart on your own hardware projects. You’ll have access to a kit as well as individualized sessions to help you master construction and coding fundamentals!</p>
```
Comment (apanpanpan) [inline index.html:204]: can you also add the hardware section to the mobile carousel below

File: index.html [hunk 2]
```diff
@@ -384,7 +391,7 @@ <h4>How can I register?</h4>
                     <h4>+</h4>
                 </div>
                 <div class="panel">
-                    <p>Registration for Technica is now open! You can register <a class="link" href=""><b>here</b></a>! TODO: HYPERLINK</p>
+                    <p>Registration for Technica is now open! You can register <a class="link" href="https://gotechnica.typeform.com/to/TcQ4ecwG"><b>here</b></a>! TODO: HYPERLINK</p>
```
Comment (apanpanpan) [inline index.html:394]: can you remove the "TODO: HYPERLINK" lol

File: index.html [hunk 3]
```diff
@@ -246,7 +253,7 @@ <h1 class="header" id="registration">Registration</h1>
             <div class="col-md order-md-1 align-self-center">
               <div class="hacker">
                 <p>Come build using hardware, software, or anything you want with other hackers! You don't have to know how to code to be a hacker. Hackers can be students, designers, or just any tech-lover out there!</p>
-                <button class="btn">Join As A Hacker</button>
+                <button class="btn" onclick="location.href='https://gotechnica.typeform.com/to/TcQ4ecwG'" type="button">Join As A Hacker</button>
```
Comment (apanpanpan) [inline index.html:256]: this location.href is cool omg i didnt know you could do that

Comment (apanpanpan) [inline index.html:256]: lets add the registration link to the main button at the top too!

File: index.html (hunk 1 above)
Comment (smohamu3) [reply index.html:204]: I'm a clown 🤦‍♀️

File: index.html (hunk 2 above)
Comment (smohamu3) [reply index.html:394]: Same here lol

**Pull request merged**